images/postgresql/00_users.sql

```diff
@@ -0,0 +1,2 @@
+CREATE ROLE daemon LOGIN PASSWORD 'password';
+CREATE ROLE website LOGIN PASSWORD 'password';
```

Comment:
I did hardcode those, I hope it's fine

Reply:
doesn't docker have some method of sharing on-the-fly generated credentials between services? Is it possible to limit the connections to localhost, either in the firewall or the `pg_hba.conf` ?

Reply:
> doesn't docker have some method of sharing on-the-fly generated credentials between services?

I can use environment variables for this yes

> Is it possible to limit the connections to localhost, either in the firewall or the pg_hba.conf ?

Technically, they are limited as one as to be inside the docker network. For now I did open port 5432 to ease debugging.

Also, the applications does not connect from localhost as they have their own container.

Reply:
I did add environment variables to this file, but hardcoded the values in `compose.yaml`. I can add a `.env` file, but I do not feel like this would be useful.